src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,88 @@
 package com.theironyard.novauc;
 
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Main {
+	public static Scanner scanner = new Scanner(System.in);
+	public static Restaurant restaurant = new Restaurant();
+
+
+
+	public static void main(String[] args) throws Exception, FileNotFoundException {
+
+
+
+
+        try {
+            restaurant = readFromFile();
+            System.out.println("Save restaurant information.");
+            System.out.println(restaurant);
+            System.out.println("Would you like to update this restaurant information? [y/n]");
```

Comment:
You need to override toString() in your Restaurant class so that it prints out nice and pretty.  But otherwise good job.